modules/tinymce/src/plugins/imagetools/test/ts/browser/ImageToolsDisabledButtonsTest.ts

```diff
@@ -0,0 +1,96 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor, TinyDom, TinySelections } from '@ephox/mcagar';
+import { Attribute, SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import PromisePolyfill from 'tinymce/core/api/util/Promise';
+import ImagePlugin from 'tinymce/plugins/image/Plugin';
+import ImageToolsPlugin from 'tinymce/plugins/imagetools/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.imagetools.ImageToolsDisabledButtonsTest', () => {
+
+  const settings = {
+    plugins: 'image imagetools',
+    toolbar: 'editimage fliph flipv rotateleft rotateright',
+    base_url: '/project/tinymce/js/tinymce',
+    height: 900
+  };
+
+  const editImageButtonSelector = '[role="toolbar"] button[title="Edit image"]';
+  const flipHorizontallyImageButtonSelector = '[role="toolbar"] button[title="Flip horizontally"]';
+  const flipVerticallyImageButtonSelector = '[role="toolbar"] button[title="Flip vertically"]';
+  const rotateCounterClockwiseImageButtonSelector = '[role="toolbar"] button[title="Rotate counterclockwise"]';
+  const rotateClockwiseImageButtonSelector = '[role="toolbar"] button[title="Rotate clockwise"]';
+
+  const assertImageToolsButtonsExist = (editor) => {
+    UiFinder.exists(TinyDom.container(editor), editImageButtonSelector);
+    UiFinder.exists(TinyDom.container(editor), flipHorizontallyImageButtonSelector);
+    UiFinder.exists(TinyDom.container(editor), flipVerticallyImageButtonSelector);
+    UiFinder.exists(TinyDom.container(editor), rotateCounterClockwiseImageButtonSelector);
+    UiFinder.exists(TinyDom.container(editor), rotateClockwiseImageButtonSelector);
+  };
+
+  const assertButtonStatus = (editor, selector: string, enabled: boolean) => {
+    UiFinder.findIn(TinyDom.container(editor), selector).map((button) => {
```

Comment:
This should just be a `getOrDie`, not a `map` as otherwise the assertion may not be working. Even better, this should be using `UiFinder.exists` or similar as we don't need to lookup the `aria-disabled` attribute and instead can use a selector.

- [ ] Use appropriate `UiFinder` assertion here instead